src/GitVersionCore/Common/IBuildServerResolver.cs

```diff
@@ -0,0 +1,7 @@
+namespace GitVersion.Common
+{
+    public interface IBuildServerResolver
+    {
+        IBuildServer GetCurrentBuildServer();
```

Comment:
With `IBuildServerResolver` being the name of the interface, wouldn't `ResolveCurrentBuildServer` or just `Resolve` be a better name for the method?

Reply:
Changed to `Resolve`